configure

```diff
@@ -16379,7 +17194,7 @@ n|N|no|No|NO)
 ;;
 esac
 
-else
+else $as_nop
```

Comment:
I suspect this is changing the behavior of the code.
I have seen cases where some doxygen features were enabled even if doxygen was not available.
While debugging this, I found out that `DX_FLAG_doc` was set and I suspect this is the cause.

The problem is: this is automatically generated.  Should I remove this patch and use older versions of GNU autoconf and GNU automake?

Reply:
>Should I remove this patch and use older versions of GNU autoconf and GNU automake?

IMO yes, hopefully the next version of autoconf fixes that we can update the version then.

Reply:
Done. I replaced the original patch with a new one that changes just a few lines.
I can't reproduce the issue anymore.